src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -55,7 +58,8 @@ protected String getEndpointPrefix() {
 
     private static final String ENABLED_SERVICE_ACCOUNT_BODY = "{"
         + " \"attributes\": { \"service\": \"true\", "
-        + "\"enabled\": \"true\"}"
+        + " \"enabled   \": \"true\"},"
+        + " \"service\": \"true\" "
```

Comment:
This should be reverted, I'd expect the attribute on line 60 to handle this case.

Reply:
https://github.com/opensearch-project/security/pull/2786/files#r1334459144